### ISSUE TITLE
Suggestion: Menace warnings

### ISSUE THREAD

**Issue by Maddremor** [lensvol/fl-quirk-master#2]
With the new [menace item system](https://www.failbettergames.com/news/fallen-london-roadmap-july-2023-and-beyond) in place, players may want to know what actions cause menace increases so that they can prepare themselves with protective item. This seems like something Quirk Master could plausibly expanded to cover. (Not pictured: the expertise to make that estimation)

**Comment by lensvol** [comment]
Yes, I had the same thought and almost started implementing this feature... But then Wiki melted down and I was no longer sure of its data integrity. As soon as I can receive a "go ahead" work will start on a prototype.
